api/envoy/api/v2/eds.proto

```diff
@@ -50,12 +50,21 @@ message ClusterLoadAssignment {
 
   // Load balancing policy settings.
   message Policy {
-    // Percentage of traffic (0-100) that should be dropped. This
-    // action allows protection of upstream hosts should they unable to
-    // recover from an outage or should they be unable to autoscale and hence
-    // overall incoming traffic volume need to be trimmed to protect them.
-    // [#v2-api-diff: This is known as maintenance mode in v1.]
-    double drop_overload = 1 [(validate.rules).double = {gte: 0, lte: 100}];
+    reserved 1;
+
+    message DropOverload {
+      // Name of the drop policy.
+      string category = 1;
+
+      // Percentage of traffic (0-100) that should be dropped.
+      double drop_percentage = 2 [(validate.rules).double = {gte: 0, lte: 100}];
+    }
+    // Action to trim the overall incoming traffic to protect the upstream
+    // hosts. This action allows protection in case the hosts are unable to
+    // recover rom an outage, or unable to autoscale or unable to handle
+    // incoming traffic volume for any reason.
+    repeated DropOverload drop_overloads = 2
+        [(validate.rules).repeated.min_items = 1];
```

Comment:
I'm not sure about this validation. If there are other, unrelated policy settings in the future we won't want to require this one.

Reply:
Removed validation.